frontend/src/components/rootModal/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import DeleteUserModal from '../deleteUserModal';
+import {
+  ModalState,
+  selectModalDetails,
+  hideModal,
+} from '../../features/modalSlice';
+
+interface ModalHash {
+  [key: string]: any;
+}
```

Comment:
if anyone can tell me the type for a generic react component, i will update this, but I need this to stop an error in line 22

Reply:
`JSX.Element` ?